code_samples/back_office/image_editor/config/packages/image_editor.yaml

```
@@ -0,0 +1,13 @@
+ezpublish:
+    system:
+        <siteaccess>:
```

How about:
```suggestion
        site:
```
so we can test the file instantly, without modifications